### ISSUE TITLE
seafile-ignore.txt currently broken with wildcards

### ISSUE THREAD

**Issue by tokred** [haiwen/seafile#1308]
Hi there,

I am a big fan of Seafile since 2.x, however recently I noticed an annoying issue:

I have the following seafile-ignore.txt in the library root directory:
````
# Lightroom Previews directories
*Previews.lrdata/
*/*Previews.lrdata/

# Lightroom Backup directory
Backups/
Backups/*

Temporary Import Data.db
Temporary Import Data.db-journal

*.swp
````
Unfortunately, Seafile seems to completely ignore all of these rules - keeps syncing vim's .swp files and Lightrooms's `<foo_catalog> Previews.lrdata` directories.

Any idea how to solve this? I am almost sure that it worked some versions ago (4.0.x or so?).

Server: Linux@4.2.3
Client: Mac@4.2.7

**Comment by FabFaeb** [comment]
I get the feeling the whole seafile-ignore.txt feature is broken. I've added several directories in my seafile-ignore file but - since the latest update - all of them are synced.

**Comment by FabFaeb** [comment]
Same problem here! Seafile just syncs everything in the library ignoring the rules in seafile-ignore.txt...
Server Linux(RPI)@4.2.3
Clients: 2x Win@4.2.8

A quick fix would be aweome!

**Comment by freeplant** [comment]
Is the problem solved with version 5.0?
